scripts/rollup/rollup.config.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+/* eslint-env node */
+
+const { readFileSync } = require('node:fs');
+const { writeFile, mkdir } = require('node:fs/promises');
+const path = require('node:path');
+const MagicString = require('magic-string');
+const { rollup } = require('rollup');
+const replace = require('@rollup/plugin-replace');
+const typescript = require('@rollup/plugin-typescript');
+const { nodeResolve } = require('@rollup/plugin-node-resolve');
+
+const pkg = JSON.parse(readFileSync(path.resolve(process.cwd(), './package.json'), 'utf-8'));
```

Comment:
Technically you can `yarn run` outside of the project root dir. Using `process.cwd()` here means that's no longer true. It looks like you're doing this so that the invoking subpackage can use this config directly, making it reusable. I think that tradeoff is fine, but this line could use a comment to clarify what's going on.

Even better - add something like `const PACKAGE_ROOT = process.cwd()`, put the comment above that line, and use `PACKAGE_ROOT` in place of `process.cwd()` elsewhere in this file.

Reply:
Done! 089cd4a65